app/src/main/java/com/davidtomas/taskyapp/features/agenda/data/agenda/repository/AgendaRepository.kt

```diff
@@ -18,28 +18,28 @@ class AgendaRepository(
     private val reminderLocalSource: ReminderLocalSource,
     private val taskLocalSource: TaskLocalSource
 ) {
-    suspend fun getAgenda(): Result<Flow<List<AgendaItem>>, DataError.Network> =
+    suspend fun getAgenda(): Result<Flow<List<AgendaModel>>, DataError.Network> =
         coroutineScope {
             val jobEvents = async { eventLocalSource.getEvents() }
             val jobTasks = async { taskLocalSource.getTasks() }
             val jobReminders = async { reminderLocalSource.getReminder() }
             val eventsResult = jobEvents.await()
             val tasksResult = jobTasks.await()
             val remindersResult = jobReminders.await()
-            val eventsFlow = eventsResult.getOrNull() ?: flow { listOf<AgendaItem>() }
-            val tasksFlow = tasksResult.getOrNull() ?: flow { listOf<AgendaItem>() }
-            val remindersFlow = remindersResult.getOrNull() ?: flow { listOf<AgendaItem>() }
-            val listAgendaItem = mutableListOf<AgendaItem>()
+            val eventsFlow = eventsResult.getOrNull() ?: flow { listOf<AgendaModel>() }
+            val tasksFlow = tasksResult.getOrNull() ?: flow { listOf<AgendaModel>() }
+            val remindersFlow = remindersResult.getOrNull() ?: flow { listOf<AgendaModel>() }
+            val listAgendaModel = mutableListOf<AgendaModel>()
             Result.Success(
                 combine(
                     eventsFlow,
                     tasksFlow,
                     remindersFlow
                 ) { events, tasks, reminder ->
-                    if (events.isNotEmpty()) listAgendaItem.addAll(events)
-                    if (tasks.isNotEmpty()) listAgendaItem.addAll(tasks)
-                    if (reminder.isNotEmpty()) listAgendaItem.addAll(reminder)
-                    listAgendaItem.toList()
+                    if (events.isNotEmpty()) listAgendaModel.addAll(events)
+                    if (tasks.isNotEmpty()) listAgendaModel.addAll(tasks)
+                    if (reminder.isNotEmpty()) listAgendaModel.addAll(reminder)
+                    listAgendaModel.toList()
```

Comment:
If the list is empty, addAll won't do anything, so the check is pointless. You could just shorten this to 
```
events + tasks + reminders
```